witness/src/Dscp/Witness/Web/Logic.hs

```diff
@@ -77,11 +77,11 @@ toTxInfo tx = TxInfo
 
 getBlocks :: WitnessWorkMode ctx m => Maybe Int -> Maybe HeaderHash -> m [BlockInfo]
 getBlocks mCount mFrom = do
-    let count = max 100 $ fromMaybe 100 mCount
+    let count = min 100 $ fromMaybe 100 mCount
     from <- maybe (runSdMRead getTipHash) return mFrom
     runSdMRead (getBlockMaybe from) >>= void . nothingToThrow BlockNotFound
     eBlocks <- runSdMRead $ getBlocksBefore count from
-    either (throwM . InternalError) (return . map (toBlockInfo False) . toList) eBlocks
+    either (throwM . InternalError) (return . map (toBlockInfo False) . reverse . toList) eBlocks
```

Comment:
Are you sure? :thinking:

Reply:
Yes, because frontend needs newest-first lists.